numba/cuda/tests/cudadrv/test_deallocations.py

```diff
@@ -38,14 +38,18 @@ def test_max_pending_bytes(self):
         try:
             # change to a smaller ratio
             config.CUDA_DEALLOCS_RATIO = max_pending / mi.total
-            self.assertEqual(deallocs._max_pending_bytes, max_pending)
+            # due to round off error (floor is used in calculating _max_pending_bytes)
+            # it can be off by 1.
+            self.assertAlmostEqual(deallocs._max_pending_bytes, max_pending, delta=1)
 
             # deallocate half the max size
+            # this will not trigger deallocation
```

Comment:
I'm confused by this comment and the one below.  Both don't trigger deallocation?